### DISCUSSION TITLE
Possible to avoid the automatic generated "Co-authored-by"?

### DISCUSSION THREAD

**Discussion by huxuan** [renovatebot/renovate#28525] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

I am running a self-hosted Renovate and authenticated with a GitHub App token. The environment varialbe  `RENOVATE_GIT_AUTHOR` is also properly configured to the the same as the user represented by the GitHub App, just like `29139614+renovate[bot]@users.noreply.github.com` to `renovate[bot]`. The dependency update works as expected. But the only problem is that every time when merging a pull request created by the bot account, a `Co-authored-by` will always generated automatically by GitHub in the commit message. I also notice that renovate repo also have similar situations, it is kind of annoy since the creator of the pull request and the author of the git commits are actually the same. Is there any approach to avoid that?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
I think this is a GitHub thing rather than Renovate, but if you find out what Renovate could do differently to avoid it, please let us know. I see the same in this repo when merging:

![image](https://github.com/renovatebot/renovate/assets/6311784/cbb99d2d-abbd-4d6f-8797-1d6d53eafdfa)

**Reply by huxuan** [reply]
Thanks for the respose, just realize that it is more like a GitHub issue. I also found a discussion here: https://github.com/orgs/community/discussions/49727, maybe we can track the problem there.

**Reply by HonkingGoose** [reply]
I searched the GitHub Docs, but did not find anything obvious on how to disable the `Co-authored-by:` part.

It looks like the only workable option is to manually edit the commit _before_ merging it. If you want you could force push a "clean commit" _after_ a merge, but I recommend _avoiding_ that.
